gui_themes.py

```diff
@@ -0,0 +1,142 @@
+import dearpygui.dearpygui as dpg
+
+
+# Source: https://github.com/hoffstadt/DearPyGui_Ext/blob/master/dearpygui_ext/themes.py
```

Comment:
Everything below is copied from this link. I've also copied the license and copyright information from there, since it's not my code.